public/modules/custom/grants_premises/src/GrantsPremisesService.php

```diff
@@ -60,31 +65,21 @@ public function processPremises(ListInterface $property, array $arguments): arra
         if (!$itemValue) {
           continue;
         }
-
-        $elementMeta = self::getMeta($itemDefinition);
-        $completeMeta = json_encode(AtvSchema::getMetaData(
-          $pageMeta, $sectionMeta, $elementMeta,
-        ), JSON_UNESCAPED_UNICODE);
-
-        // Process boolean values separately.
-        if (
-          $itemName == 'isOwnedByCity' ||
-          $itemName == 'isOthersUse' ||
-          $itemName == 'isOwnedByApplicant'
-        ) {
-          $itemValues[] = [
-            'ID' => $itemName,
-            'label' => $itemDefinition->getLabel(),
-            'value' => $itemValue,
-            'valueType' => $valueTypes['jsonType'],
-            'meta' => $completeMeta,
-          ];
-          continue;
+        $label = $itemDefinition->getLabel();
+        if (isset($webformElement['#webform_composite_elements'][$itemName]['#title'])) {
+          $label = $webformElement['#webform_composite_elements'][$itemName]['#title'];
+          if (!is_string($label)) {
+            $label = $label->render();
+          }
         }
+        $elementMeta = self::getMeta($itemDefinition);
+        $metaData = AtvSchema::getMetaData($pageMeta, $sectionMeta, $elementMeta);
+        $completeMeta = json_encode($metaData, JSON_UNESCAPED_UNICODE);
+
         // Add items.
         $itemValues[] = [
           'ID' => $itemName,
-          'label' => $itemDefinition->getLabel(),
+          'label' => $label,
```

Comment:
Pitäiskö tän mennä metadatan sisään? Nyt meta json:in sisällä on vieläkin toi suomenkielinen
![image](https://github.com/City-of-Helsinki/hel-fi-drupal-grants/assets/43133397/2fffb260-5689-40fd-82d5-b88248fff67b)